setup.cfg

```diff
@@ -0,0 +1,2 @@
+[metadata]
+description-file = README.md
```

Comment:
Looks like we're missing a newline here.

Reply:
Added a newline.